.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 
 default_language_version:
-  python: python3.7
+  python: python3
```

Comment:
What, if any, is the effect of this change?

Reply:
It wasn't letting me commit because something with the virtual environment was making it unable to find 3.7 specifically. This change just relaxes that constraint. Is there a reason to keep the constraint more specific?

Reply:
Not necessarily. The value of this was set to 3.7 a while ago, and probably could be a different version. I think it may have been choosen based on the highest supported version of Python for Beam at the time.

What was the error message?

Reply:
Something like "RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.7'"

Reply:
Do you have python3.7 installed? 

I found this thread https://github.com/pre-commit/pre-commit/issues/1375#issuecomment-603906811

Reply:
I do have 3.7 installed.

That's the same thread I used to resolve the issue.

Reply:
I see. I think it should be fine.

It is interesting though that nobody else has run into this before. Could it be related to your M1 Mac? 😆

Reply:
That's provisionally my excuse for everything from now on...

Reply:
We typically specify the lowest minor version we support when doing checks. It would be good to maintain consistency with https://github.com/apache/beam/blob/ac3766ab4847963a52d5fd5ad0c34456dbf54c65/sdks/python/tox.ini#L20 , which uses 3.7